docs/skeleton_annotation/tools.md

```diff
@@ -1,33 +1,67 @@
+- Place nodes with precise positioning
+- Connect nodes to form tree structures
+- Add branches and create complex neural tracings
 
-- `Skeleton`: Create skeleton annotations and place nodes with a left mouse click. Read more below.
+## Basic Concepts
 
-When the `Skeleton` tool is active, the following modifiers become available:
+A skeleton annotation consists of:
 
-- `Create new Tree`: Creates a new tree.
-- `Toggle single node tree mode`: This modifier makes the skeleton annotation tool create a new tree for each node instead of adding nodes to the current tree. You can use this mode to mark single objects or seeds, such as nuclei. This is also known as "Soma-clicking mode".
-- `Toggle merger mode`: This modifier activates the `Merger Mode` for the skeleton annotation tool. In merger mode, you can use skeletons to "collect" and merge volume segments from an over-segmentation. [Read more about `Merger Mode`](../proofreading/merger_mode.md).
+- **Nodes**: Individual points placed in 3D space (created with left-click)
+- **Edges**: Connections between nodes that form trees
+- **Trees**: Collections of connected nodes and edges
+- **Branch Points**: Special nodes marking where a structure splits into multiple paths
 
-![Skeleton Tool modifiers](../images/skeleton_tool_modifiers.jpeg)
+The active node is always highlighted with a circle and can be moved by dragging.
 
-- `Skeleton`: Create skeleton annotations and place nodes with a left mouse click. Read more below.
 
-When the `Skeleton` tool is active, the following modifiers become available:
+## Creating Annotations
 
-- `Create new Tree`: Creates a new tree.
-- `Toggle single node tree mode`: This modifier makes the skeleton annotation tool create a new tree for each node instead of adding nodes to the current tree. You can use this mode to mark single objects or seeds, such as nuclei. This is also known as "Soma-clicking mode".
-- `Toggle merger mode`: This modifier activates the `Merger Mode` for the skeleton annotation tool. In merger mode, you can use skeletons to "collect" and merge volume segments from an over-segmentation. [Read more about `Merger Mode`](../proofreading/merger_mode.md).
+### New Trees
+- Select the **Skeleton** tool from the toolbar
+- Create a new tree using the toolbar button or press ++c++
+- Each tree can represent a different structure or part of your annotation
+- Trees are listed and can be managed in the [right sidebar](trees_list.md)
 
-![Skeleton Tool modifiers](../images/skeleton_tool_modifiers.jpeg)
+### Branch Points
+- Mark any node as a branch point using ++b++ or the right-click menu
+- Branch points are highlighted with a distinct color
+- All branch points are stored in a first-in, first-out (FIFO) stack
+- Jump to the latest branch point using ++j++ to continue working from there
 
-## Controls & Keyboard Shortcuts for Skeleton Annotations
+### Tool Modifiers
```

Comment:
hmm, I don't really like the word modifier here. how about "Tool-specific functionality"?
I only use the term modifier in the context of keyboard shortcuts (so, shift, ctrl etc. are modifiers). that's also how we use it in the UI afaik.
I suggest to simply omit modifier in the tool descriptions:
- "New Tree". maybe add "button" or sth similar if you think that's better. but it has nothing to do with a modifier from my understanding. 
- "Single Node Tree Mode" and "Merger Mode" <-- there is already the word mode in it. "modifier" doesnt improve it imo.